cloud/docker/docker.py

```diff
@@ -92,6 +92,10 @@
       - 'alias. Use docker CLI-style syntax: C(redis:myredis).'
     default: null
     version_added: "1.5"
+  devices:
+    description:
+      - List of host devices to expose to container
+    default: null
```

Comment:
While you're rebasing, could you add `required: false` to the documentation YAML here?